kuberun/eventing/storage/main.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Sample cloud-storage is a Cloud Run service which handles events from Cloud Storage
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+)
+
+// Receive simply print the Ce-Subject header out
+func Receive(event cloudevents.Event) {
+	// do something with event.
+	fmt.Printf("Detected change in GCS bucket: %s\n", event.Subject())
```

Comment:
Is this always for a bucket? Also, I don't think "GCS" is used for branding purposes.